tests/src/main/java/net/neoforged/neoforge/debug/block/BlockTests.java

```diff
@@ -96,4 +99,27 @@ public Optional<Vec3> getRespawnPosition(BlockState state, EntityType<?> type, L
                         1, 3, 2))
                 .thenSucceed());
     }
+
+    @GameTest
+    @EmptyTemplate(floor = true)
+    @TestHolder(description = "Dead bushes should be placeable on regular terracotta (colored or not), but not on glazed terracotta")
+    static void deadBushTerracottaTest(final DynamicTest test) {
```

Comment:
Considering it is gametest only, this test can take the extendedgametesthelper as it's first argument, and the dynamic test as the second, that way the onGameTest is no longer needed.

Reply:
These are the things I wish I knew about the test framework system. Comprehensive guide when? 😄

Reply:
There was a wip one sent in the Discord channel at some point, and the "official docs" which cover this.

Reply:
Do we still need the `DynamicTest` parameter? Omitting it doesn't seem to make the test fail.

Reply:
No, you do not.